libnetwork/internal/rootlessnetns/netns_linux.go

```diff
@@ -113,7 +114,14 @@ func (n *Netns) getOrCreateNetns() (ns.NetNS, bool, error) {
 	if err != nil {
 		return nil, false, wrapError("create netns", err)
 	}
-	err = n.setupSlirp4netns(nsPath)
+	switch strings.ToLower(n.config.Network.DefaultRootlessNetworkCmd) {
+	case "slirp4netns":
```

Comment:
use constant, slirp4netns.BinaryName
also match `""` as well because we allow this in podman,buildah atm